.github/workflows/preview-pull-request.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         if: ${{ github.event.workflow_run.event == 'pull_request' && env.PR }}
         run: |
           export PATH="$(python3 -m site --user-base)/bin:${PATH}"
-          sh tools/install-dependencies.sh bikeshed wgsl
+          sh tools/install-dependencies.sh bikeshed wgsl diagrams
```

Comment:
I think we should adjust this after https://github.com/gpuweb/gpuweb/pull/3384 gets merged, as there are some aspects like diagrams might not be getting into the base from the head too. Thank you for the understanding.

Reply:
(not meaning in any sense for that PR to be a blocker on this one- could it be possible to tentatively upload preview of diagram in PR description or comment?)

Reply:
Yeah, I saw that PR, but I wanted people to be able to see the diagram here regardless. Happy to rework this once yours lands.

Reply:
This might not show in preview since the system depends on main of repo version (further, we need improvement to fetch `mmd.svg` files from head). I'll try to add this support in new one if it deploys w/o issue.

Reply:
When I look at the API side, the generated svgs are checked in, so I'm going to follow that example. Hopefully that works and I'll revert the build and action changes.

Reply:
@alan-baker , the only workaround that I can think of for the time being is embedding SVG as data image ( https://yoksel.github.io/url-encoder/ ). Please see the following line and its surrounding for reference to why things might not work out:

https://github.com/gpuweb/gpuweb/blob/main/.github/workflows/preview-pull-request.yml#L35